hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemOauth.java

```diff
@@ -90,29 +92,29 @@ public void testBlobDataContributor() throws Exception {
     // Verify Blob Data Contributor has full access to existed folder, file
 
     // READ FOLDER
-    assertTrue(fs.exists(EXISTED_FOLDER_PATH));
+    assertTrue(fs.exists(existedFolderPath));
 
     //DELETE FOLDER
-    fs.delete(EXISTED_FOLDER_PATH, true);
-    assertFalse(fs.exists(EXISTED_FOLDER_PATH));
+    fs.delete(existedFolderPath, true);
+    assertFalse(fs.exists(existedFolderPath));
```

Comment:
good time to move to the ContractTestUtils assertions assertPathExists/assertPathDoesNotExist as they provide meaningful messages/diagnostics. I really don't like assertTrue/assertFalse without messages, as they make debugging test failures much harder than they need to be